lib/tasks/rebroadcast_pac_bio_multiple_requests_run.rake

```diff
@@ -0,0 +1,74 @@
+
+
+# frozen_string_literal: true
+
+namespace :pac_bio_run do
+  desc 'Update all PacBio runs in the warehouse where there is a duplicate for the index ' \
+  '[id_pac_bio_run_lims, well_label, comparable_tag_identifier, comparable_tag2_identifier]' \
+  ' because there is more than one sequencing request for the run'
+
+  task rebroadcast_duplicates: :environment do
+    # This task redefines Receptacle to be able to only use the last active request
+    Receptacle.class_eval do
+      def most_recent_active_requests_as_target_group_by_same_source
+        requests_as_target.where(state: ['started']).group(:asset_id).having('created_at=max(created_at)')
```

Comment:
Might want to have a quick zoom about this, as it seems to be trying to do something I've struggled with for a while,  but in my experience this approach shouldn't work (although will still be fine for this purpose).

Reply:
I changed this to get the most recent request no matter the state (also the condition 'having to' I found out was wrong).